### PR TITLE
Included release note for dec-14-prd-release

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
Included release note for dec-14-prd-release

### CODE REVIEW

File: docs/whats-new/production-release-notes.md [hunk 1]
```diff
@@ -2,6 +2,63 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Production release 14 December 2022
+
+> :memo: Due to some changes we made for this release, users may experience an infinite sign-in loop when you access the TechPass portal on the staging environment. To fix this, please clear your cache.
```
Comment (JLeeChan) [inline docs/whats-new/production-release-notes.md:7]: "you may experience" sound better?

File: docs/whats-new/production-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,63 @@
...
+
+Frontend version: 1.0.0-20221205.0606   | Backend version: 1.38.0-221205.0704
```
[the remark this answers is not in the document]
Comment (JLeeChan) [reply docs/whats-new/production-release-notes.md:9]: there was a hotfix for the group name length so it bumped the versions from the deploy last week

[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md]: gotcha! will keep in mind :-) Thanks!

File: docs/whats-new/production-release-notes.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md:7]: yes, you are correct. Thanks!

Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md]: done

File: docs/whats-new/production-release-notes.md [hunk 3]
```diff
@@ -2,6 +2,63 @@
...
+
+The Bind Owners To Group Automation API returned an empty array of groups in the response body. This was redundant and we have removed it.
+
+</details>
+
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Resend user-related webhook events only to tenants for whom it failed</summary>
+
+Tenants can subscribe to webhook events such as user-invited, user-deleted and user-updated, which get triggered when users are added, deleted and removed, respectively, using TechPass Automation API, TechPass and TechBiz portals.
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md]: done, thanks Jerome!

File: docs/whats-new/production-release-notes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yilunyou) [reply docs/whats-new/production-release-notes.md]: ahh.. maybe we should update the staging versions too.. sorry for missing this

**Pull request merged**